openy_memberships.libraries.yml

```diff
@@ -0,0 +1,101 @@
+block:
+  version: 1.x
+  css:
+    theme:
+      css/paragraph_skins/block.css: {}
+
+modal:
+  version: 1.x
+  css:
+    theme:
+      css/paragraph_skins/modal.css: {}
+
+desktop-sidebar:
+  version: 1.x
+  css:
+    theme:
+      css/paragraph_skins/desktop-sidebar.css: {}
+
+library:
+  version: 1.x
+  css:
+    theme:
+      css/carnation.css: {}
+      css/paragraph_skins/mobile-sidebar.css: {}
+  js:
+    js/mobile-menu.behaviors.js: {}
+
+sidebar-menu:
+  version: 1.x
+  css:
+    theme:
+      css/paragraph_skins/sidebar-menu.css: {}
+
+button:
+  version: 1.x
+  css:
+    theme:
+      css/paragraph_skins/button.css: {}
+
+feature-card:
+  version: 1.x
+  css:
+    theme:
+      css/paragraph_skins/feature-card.css: {}
+
+banner:
+  version: 1.x
+  css:
+    theme:
+      css/paragraph_skins/banner.css: {}
+
+block3:
+  version: 1.x
+  css:
+    theme:
+      css/paragraph_skins/block3.css: {}
+
+block2:
+  version: 1.x
+  css:
+    theme:
+      css/paragraph_skins/block2.css: {}
+
+block2benefits:
+  version: 1.x
+  css:
+    theme:
+      css/paragraph_skins/block2-benefits.css: {}
+
+memberships-app:
+  version: 1.x
+  css:
+    theme:
+      css/paragraph_skins/memberships-app.css: {}
+
+simple-content:
+  version: 1.x
+  css:
+    theme:
+      css/paragraph_skins/simple-content.css: {}
+
+free-trial:
+  version: 1.x
+  css:
+    theme:
+      css/paragraph_skins/free-trial.css: {}
+
+accordion:
+  version: 1.x
+  css:
+    theme:
+      css/paragraph_skins/accordion.css: {}
```

Comment:
can we extend sass compiler with minify plugin and add `minified: true` everywhere to increase overall performance?